src/ProgramBlockEditor.js

```diff
@@ -767,6 +767,8 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
                             className={characterPositionTextInputClassName}
                             aria-label={this.props.intl.formatMessage({id:'ProgramBlockEditor.editPosition.columnPosition'})}
                             aria-disabled={this.props.editingDisabled}
+                            maxLength='2'
```

Comment:
Is this just to make them match, or are you future-proofing?  We don't have any double character column positions.

Reply:
I can see that having a `maxLength` at all is an improvement over what's deployed to production.  Can we try this with a `maxLength` of `1`?

Reply:
Good idea, I changed maxLength of column text box to be 1, and left size to be 2 for its width.

Reply:
With these changes I think this is ready for further review.